enterprise/backend/src/metabase_enterprise/stale.clj

```diff
@@ -145,3 +147,11 @@
                 (map (fn [v] (update v :model #(keyword "model" %)))))
                (t2/query (rows-query args)))
    :total (:count (t2/query-one (total-query args)))})
+
+(defsetting dismissed-collection-cleanup-banner
```

Comment:
this is an enterprise settings, so i think this is the best place to put this?

Reply:
it's appropriate for that reason, and also it's colocated with the stale feature. Seems a great spot for it.